vip-helpers/vip-elasticsearch.php

```diff
@@ -450,4 +450,16 @@ function wpcom_search_api_wp_to_es_args( $args ) {
 	}
 
     return $es_query_args;
-}
\ No newline at end of file
+}
+
+/**
+ * Filter ElasticPress index name if using VIP ES infrastructure
+ */
+function vip_elasticsearch_filter_ep_index_name( $index_name, $blog_id, $indexables ) {
+	if ( defined( 'USE_VIP_ELASTICSEARCH' ) && true === USE_VIP_ELASTICSEARCH ) {
+		$index_name = sprintf( 'vip-%s-%s-%s', VIP_GO_APP_ID, $indexables->slug, $blog_id );
```

Comment:
We are populating `VIP_GO_APP_ID` with the client site ID. This contradicts what we consider to be the App ID in the Dashboard, etc, where we use the "Parent" (`production` environment) ID as the App ID for a group of environments.

Do we want to address this here, @mjangda @hanifn?

Reply:
Yeah, good thinking. Problem is that we don't have a `VIP_GO_ENV_ID` exposed currently so we'll need to add that.

But, I think, for now, better to use `FILES_CLIENT_SITE_ID` instead.